src/main/resources/liquibase/crop_changelog/18_1_0.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="lkovacic" id="v18.1.0-1">
+        <preConditions onFail="MARK_RAN">
```

Comment:
@lik-83 should we check that the user columns exists in all tables where the column is renamed?

Reply:
I'm assuming that at that time of execution for liquibase, the columns should be exists.